web/templates/settings_overlay.hbs

```diff
@@ -198,4 +198,5 @@
             </div>
         </div>
     </div>
+    <span tabindex="0" aria-hidden="true" id="settings_end_focus"></span>
```

Comment:
I think this could be made more readily reusable if you do two things:

1. Use a class here, not an ID
2. Name the class something generic to all modals, like `return-tab-focus-to-modal-top`

That way, all it would take to use this in other modals is dropping in that `<span>` element in any given template.

Of course, you'll need to then think of or develop some reusable element to call `focus()` on in the `click_handlers.js` file, or possibly add a `data-` attribute to the span element, something like `data-top-focus-target="#settings_page"` that could be read from JS and adjusted on a per-use basis.

Reply:
Thank you @karlstolley  for the corrections. I've made the necessary changes . Kindly review